akroma-project/akroma-wallet-mobile
# feat: swipe to delete wallet

## Describe the solution

- [x] Add swipe action to delete a wallet

## How to test

1. Go home screen
2. Swipe to the left to see a delete trash icon
3. Click on it, the wallet should be deleted

## Additional Comments

- Something to take note of...

## Closes

- #162
